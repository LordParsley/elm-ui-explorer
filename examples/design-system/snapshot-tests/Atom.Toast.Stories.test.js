
    const tree = `<div>
    <div class="snapshot" id="Success">
        <div class="notification is-success" style="paddingRight:1em;width:80%;">
            <button class="delete">
            </button>
            <span>
                Toast
            </span>
        </div>
    </div>
    <div class="snapshot" id="Warning">
        <div class="notification is-warning" style="paddingRight:1em;width:80%;">
            <button class="delete">
            </button>
            <span>
                Toast
            </span>
        </div>
    </div>
    <div class="snapshot" id="Error">
        <div class="notification is-danger" style="paddingRight:1em;width:80%;">
            <button class="delete">
            </button>
            <span>
                Toast
            </span>
        </div>
    </div>
</div>`;
    test("Atom.Toast.Stories", () => {
      expect(tree).toMatchSnapshot();
    });
    