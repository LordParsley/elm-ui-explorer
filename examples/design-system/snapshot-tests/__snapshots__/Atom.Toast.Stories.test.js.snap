// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atom.Toast.Stories 1`] = `
"<div>
    <div class=\\"snapshot\\" id=\\"Success\\">
        <div class=\\"notification is-success\\" style=\\"paddingRight:1em;width:80%;\\">
            <button class=\\"delete\\">
            </button>
            <span>
                Toast
            </span>
        </div>
    </div>
    <div class=\\"snapshot\\" id=\\"Warning\\">
        <div class=\\"notification is-warning\\" style=\\"paddingRight:1em;width:80%;\\">
            <button class=\\"delete\\">
            </button>
            <span>
                Toast
            </span>
        </div>
    </div>
    <div class=\\"snapshot\\" id=\\"Error\\">
        <div class=\\"notification is-danger\\" style=\\"paddingRight:1em;width:80%;\\">
            <button class=\\"delete\\">
            </button>
            <span>
                Toast
            </span>
        </div>
    </div>
</div>"
`;
